extension_jwt.go

```diff
@@ -0,0 +1,155 @@
+package helix
+
+import (
+	"encoding/base64"
+	"fmt"
+	"time"
+
+	"github.com/golang-jwt/jwt"
+)
+
+// RoleType The user role type
+type roleType string
+
+// Types of user roles used within the JWT Claims
+// TODO expose these when helix supports them
```

Comment:
Do we still need this TODO?

Reply:
So user roles are largely what developers can use to work the type of viewer by extracting Twitch claims from a JWT token sent  by the twitch extension.

Currently Helix only seems to utilise the 'External' user role, however developers could pratically make use of the other user roles avaliable.

Because of the lack of use of other user roles by Helix, i was hesitant to expose them.